content/docs/search-data.md

```diff
@@ -1,7 +0,0 @@
----
-layout: "search-data"
-outputs: ["json"]
-exclude_from_pulumi_search_index: true
```

Comment:
We should do a scrub of all files and remove use of `exclude_from_pulumi_search_index`.